protobuf/src/test/scala/magnolify/protobuf/test/ProtobufTypeSpec.scala

```diff
@@ -37,20 +37,16 @@ import scala.reflect._
 object ProtobufTypeSpec extends MagnolifySpec("ProtobufRecordType") {
   private def test[T: ClassTag: Arbitrary: Eq, U <: Message: ClassTag](
     implicit tpe: ProtobufType[T, U],
-    eqMessage: Eq[U] = Eq.instance[U] { (first, second) =>
-      first.getDescriptorForType == second.getDescriptorForType &&
-      (first.getAllFields.asScala == second.getAllFields.asScala)
-    }
+    eqt: Eq[T],
+    eqm: Eq[U] = Eq.instance[U](_.toByteString == _.toByteString)
```

Comment:
Since we can't chain implicits for leave fields, might as well compare serialized bytes, which is pretty deterministic for protos.